getyourdata/data_request/templates/data_request/mail/request.html

```diff
@@ -41,8 +41,12 @@
 
         }
 
+        .datarequest-content3 {
+            padding-top: 1cm;
+        }
+
         .datarequest-footer {
-            padding-top: 2cm;
+            padding-top: 1cm;
```

Comment:
'cm' units are not recommended to be used; using px or em would be better:
https://www.w3.org/Style/Examples/007/units.en.html